src/desktop/LayerSurface.cpp

```diff
@@ -4,6 +4,7 @@
 #include "../protocols/LayerShell.hpp"
 #include "../protocols/core/Compositor.hpp"
 #include "../managers/SeatManager.hpp"
+#include "managers/input/InputManager.hpp"
```

Comment:
why? ../Compositor includes this

Reply:
probably auto included by clangd

Reply:
ah yeah turn that shit off it sucks in hl